CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+## Breaking Changes
+
+- the `-` is no longer a valid part of identifiers
```

Comment:
maybe also mention that the syntax for constructing binaries changed. It might not be obvious from just disallowing `-` in identifiers.

Reply:
Good call!